source/module_io/write_Vxc.hpp

```diff
@@ -149,6 +149,25 @@ namespace ModuleIO
         return e;
     }
 
+    template<typename T>
+    std::vector<double> orbital_energy(
+        const int ik,
+        const int nbands,
+        const std::vector<T>& mat_mo,
+        const Parallel_2D& p2d)
+    {
+        std::vector<double> e(nbands, 0.0);
```

Comment:
I will suggest using "defensive coding", check nbands>=0 when it comes into this function.